Google.PowerShell/Common/ActiveUserToken.cs

```diff
@@ -12,37 +14,131 @@ public class ActiveUserToken
         /// <summary>
         /// The current active user that corresponds to this token.
         /// </summary>
-        private string activeUser = CloudSdkSettings.GetSettingsValue("account");
+        private string activeUser;
 
-        /// <summary>Gets or sets the access token issued by the authorization server.</summary>
-        [Newtonsoft.Json.JsonPropertyAttribute("access_token")]
-        public string AccessToken { get; set; }
+        /// <summary>
+        /// This function will return the current active user.
+        /// </summary>
+        internal Func<string> getActiveUser;
 
-        /// <summary>Gets or sets the lifetime in seconds of the access token.</summary>
-        [Newtonsoft.Json.JsonPropertyAttribute("expires_in")]
-        public Nullable<long> ExpiresInSeconds { get; set; }
+        /// <summary>The access token issued by the authorization server.</summary>
+        public string AccessToken { get; internal set; }
 
-        /// <summary>The date and time that this token was issued. <remarks>
+        /// <summary>
+        /// The date and time that this token was issued.
+        /// This is set in UTC time.
         /// It should be set by the CLIENT after the token was received from the server.
-        /// </remarks> 
         /// </summary>
-        public DateTime Issued { get; set; }
+        public DateTime ExpiredTime { get; internal set; }
```

Comment:
If it is instead, the time when the token expires, perhaps "Expiration" or "Expiry" would be clearer?